builder/builder.go

```diff
@@ -774,6 +860,16 @@ func (b *Builder) AddBundles(bundles []string, allLocal bool, allUpstream bool,
 		if err != nil {
 			return errors.Wrapf(err, "Failed to read local bundles dir: %s", b.LocalBundleDir)
 		}
+		// handle packages defined in local-packages, if it exists
+		pwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		localPackagesFile := filepath.Join(pwd, "local-packages")
```

Comment:
On top of making it configurable, pulling it out into `b.LocalPackagesFile` also lets us not have this file name hardcoded in multiple places.

Reply:
@matthewrsj - Even if you don't make this configurable yet, you could still do the `b.LocalPackagesFile` part so the name isn't hard-coded multiple places.

Reply:
ok, I'll do that. I didn't do that because of how long the builder struct already was, but it'll make it easier to make configurable in the future.

Reply:
Done

Reply:
Did you skip this one? I see the one in `isLocalPackagePath` got converted, but this one still shows as "local-path"

Reply:
Oops, done.